pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -894,14 +920,20 @@ private List<Metrics> getIgnoredCommandMetrics(String advertisedBrokerAddress) {
         return List.of(metric);
     }
 
-    private void monitor() {
+    @VisibleForTesting
+    protected void monitor() {
         try {
             initWaiter.await();
 
             // Monitor role
             // Periodically check the role in case ZK watcher fails.
             var isChannelOwner = serviceUnitStateChannel.isChannelOwner();
             if (isChannelOwner) {
+                // System topic config might fail due to the race condition
+                // with topic policy init(Topic policies cache have not init).
+                if (!configuredSystemTopics) {
```

Comment:
Why this change can fix this issue?

Reply:
Can you enable the topic-level policies on the integration test(`ExtensibleLoadManagerTest`) in this PR? I see it has been disabled by #21231